### ISSUE TITLE
It doesn't work with Gradle 5

### ISSUE THREAD

**Issue by eungjun-yi** [ksoichiro/gradle-console-reporter#12]
```
$ ./gradlew jacocoTestReport

FAILURE: Build failed with an exception.

* What went wrong:
Could not find method execute() for arguments [] on task ':reportJacoco' of type com.github.ksoichiro.console.reporter.ReportJacocoTask.
```

**Comment by simartn** [comment]
I hope this gets fixed here. If you only use jacoco coverage reports you maybe can switch to the less powerful plugin https://github.com/jansauer/gradle-print-coverage-plugin

**Comment by eungjun-yi** [comment]
@simonnagl de.jansauer.printcoverage works well for me. Thank you for letting me know!

**Comment by ksoichiro** [comment]
Sorry for my late reply.
I confirmed the issue.
It seems the commit a9afddf (which has not been released) has already fixed the issue so I've just released v0.6.0.
